database/elasticSearch.js

```diff
@@ -1,5 +1,8 @@
-const v8 = require('v8');
-const db = require('./index');
+var elasticsearch = require('elasticsearch');
+
+var client = new elasticsearch.Client({
+  host: 'localhost:9200'
```

Comment:
You might want to source `localhost` and `9200` from an environment variable. Maybe something like:
process.env.ES_HOST
process.env.ES_PORT